instagram/src/components/CommentSection/CommentContainer.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import './CommentContainer.css';
+import PropTypes from 'prop-types';
+
+class CommentContainer extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            commentData: this.props.postData.comments,
+            commentValue: ''
+        }
+
+        this.newCommentSubmit = this.newCommentSubmit.bind(this);
+        this.newCommentChange = this.newCommentChange.bind(this);
+        this.checkLiked = this.checkLiked.bind(this);
+    }
+
+    newCommentSubmit(event) {
+        let newComment = [...this.state.commentData]
+        let username = localStorage.getItem('username');
+
+        event.preventDefault();
+
+        newComment.push({id: (Date.now()).toString(), username: username, text: this.state.commentValue});
+        this.setState({commentData: newComment})
+    }
+    
+    newCommentChange(event) {
+        this.setState({commentValue: event.target.value});
+    }
+
+    checkLiked() {
+        if(this.props.postData.liked){
+            return <i onClick={(event) => this.props.likedHandler(event, this.props.postData.id)} className="fas liked fa-heart fa-2x"></i>
+        }
+        else {
+            return <i onClick={(event) => this.props.likedHandler(event, this.props.postData.id)} className="far fa-heart fa-2x"></i>
+        }
+    }
+    
```

Comment:
The logic in your checkLiked function can be done with a ternary inside of the component itself, look below for suggested change.